demo/vagrant/Vagrantfile

```diff
@@ -58,6 +58,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         node.vm.synced_folder "../../release", "/release", id: "release", :nfs => true, :mount_options => ['nolock,vers=3,tcp']
       end
 
+      # copy ceph config override file
+      node.vm.provision "file", source: "ceph-override.conf", destination: "/tmp/rook-conf/ceph-override.conf"
```

Comment:
i wanted vagrant to not get in the way of `rookd` making/owning that directory, but i'll give it a shot.  it would be cleaner to just use `/var/lib/rook` as that would remove the extra bind mount too.